eclair-core/src/test/scala/fr/acinq/eclair/integration/basic/TwoNodesIntegrationSpec.scala

```diff
@@ -0,0 +1,96 @@
+package fr.acinq.eclair.integration.basic
+
+import fr.acinq.bitcoin.scalacompat.{ByteVector32, SatoshiLong}
+import fr.acinq.eclair.channel.{DATA_NORMAL, NORMAL}
+import fr.acinq.eclair.integration.basic.fixtures.TwoNodesFixture
+import fr.acinq.eclair.testutils.FixtureSpec
+import fr.acinq.eclair.{BlockHeight, MilliSatoshiLong}
+import org.scalatest.TestData
+import org.scalatest.concurrent.IntegrationPatience
+import scodec.bits.HexStringSyntax
+
+
+/**
+ * This test checks the integration between Channel and Router (events, etc.)
+ */
+class TwoNodesIntegrationSpec extends FixtureSpec with IntegrationPatience {
```

Comment:
So these integration tests should in theory be much slower than our current integration tests if each tests re-does channel setup, right? How much of an issue is it in practice? It's really a pain that our current integration tests depend on the previous tests, but last time we tried to get rid of it the performance was too bad...

Reply:
I don't think they will be slower, quite the opposite. Our standard integration tests use a real bitcoind, which takes a long time to start up and initialize. Those tests are really light, and since they offer foolproof way of cleaning up fixtures, we don't end up consuming huge amount of resources and can parallelize better.